### ISSUE TITLE
Support specifying config file format in initialization wizard

### ISSUE THREAD

**Issue by irevenko** [eslint/eslint#13795]
**The version of ESLint you are using.**
v7.12.0

**The problem you want to solve.**
When i execute ```eslint --init``` i want to specify the ```.eslintrc``` format and not getting asked config file format question.

**Your take on the correct solution to problem.**
For instance: ```eslint --init --rc-ext=json```

**Comment by mdjermanovic** [comment]
Hi @irevenko, thanks for the enhancement proposal!

Can you please provide more context, is there a use case where it would be particularly useful to provide config file format as a CLI argument instead of selecting it from the list?

**Comment by irevenko** [comment]
@mdjermanovic Flags usually alow to build a config more quickly. 
**My use case**: I'm building the CLI for generating JavaScript projects and ESLint is one of the many options.
It's easier to update ```JSON``` file with Node rather than ```js``` or ```yaml``` and i want to be ```JSON``` extesion set by default and not getting asked User config file format question

**Comment by nzakas** [comment]
We are moving to a config system that only supports JavaScript files, so I don’t think it makes sense to make this change at this point.
